kpops/component_handlers/kafka_connect/model.py

```diff
@@ -95,16 +92,65 @@ def serialize_model(
         return {by_alias(self, name): to_str(value) for name, value in result.items()}
 
 
+class ConnectorState(StrEnum):
+    RUNNING = auto()
+    PAUSED = auto()
+    STOPPED = auto()
+
+    @property
+    def api_value(self) -> str:
+        return self.value.upper()
+
+
 class ConnectorTask(BaseModel):
     connector: str
     task: int
 
 
-class KafkaConnectResponse(BaseModel):
+class CreateConnector(BaseModel):
+    config: KafkaConnectorConfig
+    initial_state: ConnectorState | None = None
+
+    @computed_field
+    @property
+    def name(self) -> str:
+        return self.config.name
+
+    @field_serializer("initial_state")
+    def serialize_initial_state(self, initial_state: ConnectorState) -> str:
+        return initial_state.api_value
+
+
+class ConnectorStatus(BaseModel):
+    state: ConnectorState
```

Comment:
Pretty sure for the API response the state can also be failed

Reply:
added
you're right, there's an example in the Kafka Connect docs, however I cannot find a proper schema for the REST API